### DISCUSSION TITLE
Page loads and scrolls down automatically

### DISCUSSION THREAD

**Discussion by fiatjaf** [srid/emanote#417] [Q&A] [answered]
**Describe the bug**
After loading a page we're automatically scrolled down, always by the same amount.

**To Reproduce**
You can try it here: https://fiatjaf.com/drivechain.html
But it affects any page that can be scrolled.

**Screenshot**
https://user-images.githubusercontent.com/1653275/216092967-ba216a43-cb64-4aad-9a77-962926c9972b.mp4

**Additional Context**
I don't know if this is related to something I did, but I didn't really do anything besides setting the thing to render like Neuron.

My source code is here: https://github.com/fiatjaf/z

**Comment by fiatjaf** [comment]
Similar to https://github.com/srid/emanote/issues/262

**Comment by srid** [comment, marked as answer]
@fiatjaf The `emanote gen` generated static site should not have any special JavaScript (unlike `emanote run`), so it shouldn't be Emanote that is doing the scrolling here. 

It seems like it is that commenting widget that is doing the scrolling?

**Reply by srid** [reply]
ie., culprit seems to be https://github.com/fiatjaf/z/blob/19e1be74344ce1637ebcc36cee508640f6a6fb31/templates/hooks/after-note.tpl#L11

if you remove that, the problem should go away.

**Comment by fiatjaf** [comment]
Oh, I totally forgot I had that widget, sorry. That should have been the first place I would look.
